index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>Product Landing Page - freecodecamp</title>
+</head>
+
+<body>
+  <header id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="logo" class="logo">
+      <ul>
+        <a class="nav-link" href="#features">Features</a>
+        <a class="nav-link" href="#how-it-work">How It Works</a>
+        <a class="nav-link" href="#pricing">Pricing</a>
+      </ul>
+    </nav>
+  </header>
+
+  <div id="container">
+    <section class="message-box">
+      <h1>Handcrafted, home-made asterpieces</h1>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <input id="email" type="email" name="email" placeholder="example@mail.com">
+      <input id="submit" type="submit" value="Get Started" class="btn"></input>
+      </form>
+    </section>
+
+    <section id="features">
+      <div class="feature">
+        <i class="fab fa-free-code-camp fa-3x"></i>
+        <div>
+          <h2>Premium Materials</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+            purchase.</p>
+        </div>
+      </div>
+      <div class="feature">
+        <i class="fas fa-shipping-fast fa-3x"></i>
+        <div>
+          <h2>Fast Shipping</h2>
+          <p>We make sure you receive your trombone as soon as we have finished making it. We also provide free returns
+            if you are not satisfied.</p>
+        </div>
+      </div>
+      <div class="feature">
+        <i class="fas fa-battery-full fa-3x"></i>
+        <div>
+          <h2>Quality Assurance</h2>
```

Comment:
There shouldn't be a section tag without a heading tag for accessibility reasons. Kindly fix this.

Reply:
please ensure your comments are as polite as possible and should be as suggestions instead of orders in order to communicate better with the student and you give a detailed, clear and easy to understand